lib/mdl.rb

```diff
@@ -66,7 +66,8 @@ def self.run(argv = ARGV)
           Dir.chdir(filename) do
             cli.cli_arguments[i] =
               Mixlib::ShellOut.new("git ls-files '*.md' '*.markdown'")
-                              .run_command.stdout.lines.map(&:strip)
+                              .run_command.stdout.lines
+                              .map { |m| "#{filename}/#{m.strip}" }
```

Comment:
as a relatively new maintainer, I'm not entirely certain what the common case here is, but we should probably strive to use `File.join` instead of manual dir strings, for cross-platform goodness...